### PR TITLE
Add hello_flutter and make it work on Fuchsia

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde

File: sky/engine/BUILD.gn [hunk 1]
```diff
@@ -22,6 +22,13 @@ config("config") {
   cflags = [
     "-Wno-inconsistent-missing-override",
   ]
+
+  if (is_fuchsia) {
+    # TODO(abarth): Remove once the kernel support unmapping memory properly.
+    defines = [
+      "MEMORY_TOOL_REPLACES_ALLOCATOR",
```
Comment (chinmaygarde) [inline sky/engine/BUILD.gn]: This reminds me to remove the flag on iOS as well and get rid of the `mprotect`s to see if I can get any measurable performance difference using the different allocators.

**Comment by chinmaygarde** [comment]
lgtm

File: sky/engine/BUILD.gn (hunk 1 above)
Comment (abarth) [inline sky/engine/BUILD.gn]: IMHO, we should just rip out partition alloc entirely.  I doubt it's giving us much of any benefit.

**Pull request merged**